### PR TITLE
Team

### PR DESCRIPTION
**Pull request by Rostammahabadi**
created team which passes all tests

### CODE REVIEW

File: lib/team.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require 'csv'
+
+class Team
+
+  @@all = nil
```
Comment (Rostammahabadi) [inline lib/team.rb:5]: can be consolidated in an inheritance later down the line

File: lib/team.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  def self.all
+    @@all
+  end
```
Comment (Rostammahabadi) [inline lib/team.rb:9]: review above comment

File: lib/team.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  def self.from_csv(file_path)
+    csv = CSV.read("#{file_path}", headers: true, header_converters: :symbol)
+    @@all = csv.map do |row|
+      Team.new(row)
+    end
+  end
```
Comment (Rostammahabadi) [inline lib/team.rb:16]: Can be added into inheritance. functionality passes tests and creates objects which are stored into @@all which is an array

File: lib/team.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  attr_reader :team_id, :franchise_id, :team_name, :abbreviation, :stadium, :link
+
+  def initialize(team_info)
+    @team_id = team_info[:team_id].to_i
+    @franchise_id = team_info[:franchiseid].to_i
+    @team_name = team_info[:teamname]
+    @abbreviation = team_info[:abbreviation]
+    @stadium = team_info[:stadium]
+    @link = team_info[:link]
+  end
```
Comment (Rostammahabadi) [inline lib/team.rb:27]: followed ruby convention for the variable naming

File: lib/team.rb (hunk 1 above)
Comment (perryr16) [reply lib/team.rb:5]: agreed

File: lib/team.rb (hunk 3 above)
Comment (perryr16) [reply lib/team.rb:16]: +1, prevents the need to run the from_csv everytime we want to access the collection

File: lib/team.rb (hunk 4 above)
Comment (perryr16) [reply lib/team.rb:27]: we all need to be careful when calling headers like :franchiseid instead of :franchise_id

**Review by perryr16 — COMMENTED** [review]
looks functional

File: lib/team.rb (hunk 1 above)
Comment (stellakunzang) [reply lib/team.rb:5]: bingo

**Comment by foleymichelle9** [comment]
awesome looks great!

File: lib/team.rb (hunk 4 above)
Comment (stellakunzang) [reply lib/team.rb:27]: @perryr16 I'm not sure how to be careful. Do you mean changing the instance variable to follow the correct convention?

**Comment by stellakunzang** [comment]
Excellent! let's merge!

**Pull request merged**